shared/constants/bridge.ts

```diff
@@ -11,3 +11,8 @@ export const ALLOWED_BRIDGE_CHAIN_IDS = [
   CHAIN_IDS.LINEA_MAINNET,
   CHAIN_IDS.BASE,
 ];
+
+export const TRUSTED_BRIDGE_SIGNER =
```

Comment:
Following the comment about a potential origin constant, should we also extend this into a map such as `TRANSACTION_VERIFICATION_SIGNER_BY_ORIGIN`?

In that case, we'd likely want to move these constants to a more generic file.

Reply:
If we had a truly generic solution (one that Uniswap is supposedly working on a proposal for). I think a mapping like that would fit really cleanly into some decentralized registry like ENS.
I have this as one of the potential solutions identified in this [solutions doc](https://docs.google.com/document/d/1yEH6VKyZf6sauMHTUs9WWZ7cv_d2QfE6q44B0qNpKdQ/edit), however the goal of this PR is to stop the [bleed](https://dune.com/queries/3733704/6279934) and iterate on a solution that further generalizes this concept.

Reply:
Similar to what I have written https://github.com/MetaMask/metamask-extension/pull/25047#discussion_r1635563623
There is now a map of trusted bridge signers broken down by experiences type